content/realtime/channel-metadata.textile

```diff
@@ -18,98 +17,27 @@ jump_to:
-
-h3(#reactor). Reactor rules
-
-You can associate "Reactor rules":https://www.ably.io/reactor with metachannels in the same way as for regular channels; configure these using the "Message":/realtime/messages source and use a @channelFilter@ of, for example, @^\[meta\]log@. Normally the @channelFilter@ regex will only match normal, non-meta channels; however, a regex of the form @^\[qualifier\]pattern@ is special-cased to match channels with that qualifier. The @pattern@ is a regex as normal.
-
-If you want webhooks for channel lifecycle events, creating a "channel lifecycle webhook":/general/events#sources is an easy alternative to creating a message rule for the @[meta]channel.lifecycle@ metachannel; such webhooks will have simpler POST bodies as the lifecycle events won't be wrapped in "@Message@":/realtime/types#message objects.
+1. "Metachannels":/realtime/metachannels - Metachannels are special channels that provide metadata about all channels associated with an Ably application, this includes metadata associated with channel and connection lifecycle.
+2. "Inband channel occupancy events":/realtime/inband-occupancy - Inband occupancy events are occupancy events that are present on a user-created channel. 
+3. "Channel Status API":/rest/channel-status - The Channel Status API is a REST API that allows users to query a channel to retrieve channel metadata. It also allows you to "enumerate":/rest/channel-status#enumeration-rest all the active channels in a particular app. The details of the Channel Status API are covered in the "Channel Status documentation":/rest/channel-status. It is not recommended you poll for status updates as this is inefficient.
 
 h2(#use-cases). Use cases
 
-Having access to channel metadata can provide numerous benefits. In a scenario where the number of subscribers of a channel goes well beyond a hundred, usage of other options such as "presence":/core-features/presence becomes less effective leading to an unexpected n-squared problem if all of the clients are subscribed to presence. You could instead make use of our "channel metadata":#metachannels to check the number of active subscribers.
+Having access to channel metadata can provide numerous benefits. In a scenario where the number of subscribers to a channel exceeds 100, usage of other options such as "presence":/core-features/presence becomes less effective, as there is n-squared loading if all of the clients subscribe to presence. You can alternatively use "metachannels":/realtime/metachannels to check the number of active subscribers.
```

Comment:
"You can alternatively use "metachannels":/realtime/metachannels **or inband occupancy** ..."

Reply:
Fixed to include both options.